evm_loader/program/src/entrypoint.rs

```diff
@@ -791,16 +791,8 @@ fn process_instruction<'a>(
             (&mut data[valids_begin..valids_end]).copy_from_slice(&valids);
 
             Ok(())
-        }
-
-        EvmInstruction::Write |
-        EvmInstruction::Cancel |
-        EvmInstruction::Finalise |
-        EvmInstruction::CreateAccountWithSeed |
-        EvmInstruction::ExecuteTrxFromAccountDataIterative |
-        EvmInstruction::PartialCallFromRawEthereumTX |
-        EvmInstruction::Continue
-        => Err!(ProgramError::InvalidInstructionData; "Deprecated instruction"),
```

Comment:
I think it would be better to keep deprecated cases and list them explicitly

Reply:
This part is a bit strange and inconsistent. Half of these cases will never happen.

Deprecated instructions already listed in the `instruction.rs`. 
Over time, there number will increase. 
And keeping hundredth deprecated instructions in this match will generate a lot of noise.

A agree about moving each case into a separate method, but it is out of scope of this pull request.